x-pack/filebeat/input/unifiedlogs/input.go

```diff
@@ -222,8 +224,17 @@ func (input *input) processLogs(stdout io.Reader, pub inputcursor.Publisher, log
 		err           error
 	)
 
-	for scanner.Scan() {
-		line = scanner.Text()
+	for {
+		line, err = reader.ReadString('\n')
+		if err != nil {
+			if err == io.EOF {
+				return nil
+			}
+			return err
```

Comment:
Shouldn't you add an error here like there was for `err = scanner.Err(); err != nil`?